### PR TITLE
API calls - GET and POST

### PR DESCRIPTION
**Pull request by Ellon-M**
## Changes in this PR

- Hit the Leaderboard API to get the unique game ID.
- Used the unique ID to add scores of different users into the API using a POST request.
- Accessed each user's scores using GET requests to the API.
- Attached event listeners to the DOM to perform the API operations upon user interaction with the page.

### CODE REVIEW

**Comment by omarsalem7** [comment]
Hi @Ellon-M 😊, 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 💀 

You have submitted a project, that missing the [Future Features](https://github.com/Ellon-M/leaderboard/tree/api-methods#future-features) section in your README. Kindly add it which includes 1-3 potential sentences about the feature you will work on them in the future.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@ <h1 class="main-heading">Leaderboard</h1>
     <section class="scores-table">
       <div class="scores-top">
         <h3 class="scores-heading"></h3>
-        <a href="" class="refresh-btn">Refresh</a>
+        <button class="refresh-btn">Refresh</button>
```
Comment (fobadara) [inline src/index.html:20]: [REQUIRED] Kindly ensure that the refresh button is working as expected. The scores are not displayed when I click on it.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Ellon-M, 
Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good job doing the extra job of validating the inputs

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (Ellon-M) [reply src/index.html:20]: Understood. Working on that right now.

**Review by TadesseDev — APPROVED** [review]
### Hi @Ellon-M 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
ST3D


## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed ✅
## Optional suggestions
- [Optional] Your scores are empty on page load until the user clicks on refresh. For better user experiences it would be wise to have your board ready on the initial load.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**